clients/java/src/main/java/io/camunda/zeebe/client/impl/search/response/OperationImpl.java

```diff
@@ -30,8 +30,8 @@ public class OperationImpl implements Operation {
   public OperationImpl(final OperationItem item) {
     id = item.getId();
     batchOperationId = item.getBatchOperationId();
-    type = item.getType().toString();
-    state = item.getState().toString();
+    type = (item.getType() == null) ? null : item.getType().toString();
```

Comment:
💭 minor/suggestion: 
We can use here: 
`Optional.ofNullable(item.getType()).map(Enum::toString).orElse(null)` 

for setting `type` and `state`.